src/styles/app.css

```diff
@@ -0,0 +1,14 @@
+.app {
+  display: flex;
+  flex-direction: column;
+}
+.nasa-logo {
+  margin: auto;
+   width: 50%;
+}
+
+/* @media screen and (min-width: 1024px) {
+  .nasa-logo {
+    width: 30%;
+  }
+} */
```

Comment:
always remove comments before you push. If there's something you wanted to do and didn't have time, add a todo. eg:
```
// make nasa-logo responsive
```